NEWS.md

```diff
@@ -1,3 +1,28 @@
+# DataFrames v1.0 Release Notes
+
+## Breaking changes
+
+* No breaking changes are planned for v1.0 release
+
+## Bug fixes
+
+* fixed bug when displaying floating point columns with eltype turned off in
```

Comment:
Shouldn't we make a 0.22.1 release with small fixes like this before moving on to 1.0? It will take some time until packages and users have tested 0.22 to make sure we're totally OK with the API, and during that period it would be nice to be able to fix small bugs.

Reply:
We will make 0.22.1 as a separate branch into which I will cherry-pick the commits that should go there. This is what we did in 0.21 release.

Reply:
But I thought that making 0.22.1 only with this minor printing fix was not justified

Reply:
As you like.

Reply:
Actually I came to the conclusion that after merging #2553 we can make a patch release with everything merged to that point.

Reply:
So at any rate this bug won't appear in 1.0 release notes since it will have been fixed in 0.22.1, right?

Reply:
OK - removed